### DISCUSSION TITLE
What counts as a "website" and what as a brand/service/...?

### DISCUSSION THREAD

**Discussion by Andre601** [simple-icons/simple-icons#6695] [General]
In the contributing guidelines is the rule, that logos of website should be below 500k in the Alexa ranking.

My question now is: When does a site count as website and when is it a brand, service or similar?

For example: Would Discord count as a website, or would the fact of them offering an online chat change this perspective?
I ask because I consider PRing the revolt.chat logo to Simple Icons, but I'm not quite sure if you would count it as website or a chat or something similar...

Before you ask: The Alexa rank is not below 500k. It currently sits at around 705k

**Comment by jorgeamadosoria** [comment]
> In the contributing guidelines is the rule, that logos of website should be below 500k in the Alexa ranking.
> 
> My question now is: When does a site count as website and when is it a brand, service or similar?
> 
> For example: Would Discord count as a website, or would the fact of them offering an online chat change this perspective? I ask because I consider PRing the revolt.chat logo to Simple Icons, but I'm not quite sure if you would count it as website or a chat or something similar...
> 
> Before you ask: The Alexa rank is not below 500k. It currently sits at around 705k

so, as I see it, it depends heavily on the most available (or convenient) way you have to measure popularity.

For example, Discord. It is a website, and also an app. So you could try and measure Alexa for the website, and if it is below 500k, you could try and measure downloads of the Discord app, specially if compared with similar apps.

So, basically, metrics are there for you to pick the most convenient one that applies to your case, and if none do, then you are welcome to make a compelling case to support your request, which will be evaluated. Who knows, you may end up setting a new precedent (although the most common case is that the request will be rejected).

Revolt has several metrics that can be used: As a website, as a Github project, and as an app. 
Alexa rank is above 500k, so that's not going to work.
As a Github project, none of the repos has cracked above 1k starts, so that's out too.
If you could mount a case that shows that Revolt's app downloads are significant, specially when compared with similar apps on its niche, then we may evaluate that and decide to add it.

**Reply by Andre601** [reply]
Thanks for the detailed response.

I don't really know how - or more precisely *where* - I could check for the overall downloads the app has. They do offer a Desktop app, but it afaik is done through GitHub releases which do not give any real metrics in the end.
On the bright side does the Alexa rank show a significant grow in the past 90 days (Or more accurately 15-30 days) as it was ~900k 90 days ago and has reached 705k as of right now[^1].
So maybe it will grow further in the future days to once reach a rank below 500k.

[^1]: https://autumn.revolt.chat/attachments/h_eDAUTxLEnrDL2PQppZ5wsizvmSevW6DRuTG4RH2h/image.png

**Reply by jorgeamadosoria** [reply]
> Thanks for the detailed response.
> 
> I don't really know how - or more precisely _where_ - I could check for the overall downloads the app has. They do offer a Desktop app, but it afaik is done through GitHub releases which do not give any real metrics in the end. On the bright side does the Alexa rank show a significant grow in the past 90 days (Or more accurately 15-30 days) as it was ~900k 90 days ago and has reached 705k as of right now[1](#user-content-fn-1-87c2757de0d94b0389fc2363d4ffdcb9). So maybe it will grow further in the future days to once reach a rank below 500k.
> 
> ## Footnotes
> 1. https://autumn.revolt.chat/attachments/h_eDAUTxLEnrDL2PQppZ5wsizvmSevW6DRuTG4RH2h/image.png [↩](#user-content-fnref-1-87c2757de0d94b0389fc2363d4ffdcb9)

that is definitely a possibility. It wouldn't be the first time it happens, that's for sure.
You can keep watch, and as soon as it breaks 500k, then we can revisit Revolt.chat!

**Comment by 8KCool** [comment]
I don't really know how

**Comment by adamrusted** [comment]
Closing, as our popularity metrics are currently under review in https://github.com/simple-icons/simple-icons/pull/10114, which will hopefully better define what we're including in this project.
